plugins/module_utils/data_set.py

```diff
@@ -1850,14 +1855,386 @@ def __init__(
         self.sms_management_class = sms_management_class
         self.volumes = volumes
         self.is_gds_active = False
+        self.is_cataloged = False
+
         # If name has escaped chars or is GDS relative name we clean it.
-        self.name = DataSet.escape_data_set_name(self.name)
+        # self.name = DataSet.escape_data_set_name(self.name)
         if DataSet.is_gds_relative_name(self.name):
             try:
                 self.name = DataSet.resolve_gds_absolute_name(self.name)
```

Comment:
Here also to use the `is_gds_active` as a False al the time if resolve the gdg set `is_gds_active` to True

Reply:
Nice catch